cypress/app/app.js

```diff
@@ -513,6 +521,95 @@ app.get('/asset-456.js', (req, res) => {
   res.send(`console.log('456');`);
 });
 
+app.get('/swap-method', (req, res) => res.send(`
+  <html>
+    <head>
+      <title>Button links</title>
+      <script src="/${frontendJS}" type="application/javascript"></script>
+      <link rel="stylesheet" href="/${mainCSS}">
+    </head>
+    <body>
+      ${menu}
+      ${layout(`
+        <h1>Button links</h1>
+        <div class="content">Not swapped</div>
+        <button data-swap-method="post" data-swap-action="/swap-method-post" data-swap-body='{"name":"charles"}'>Post Data</button>
+        <button data-swap-method="put" data-swap-action="/swap-method-put" data-swap-body='{"name":"john"}'>Put Data</button>
+        <button data-swap-method="post" data-swap-action="/swap-method-post" data-swap-body='{"name": "martin"}' data-swap=".content">Swap Data</button>
+        <button data-swap-method="post" data-swap-action="http://localhost:8888/swap-method-other" data-swap=".content">Swap Other Domain</button>
+        <button data-swap-method="post" data-swap-action="/swap-method-post" data-swap-body='{"name":"jones"}' data-swap-inline=".content">Swap Data Inline</button>
+        <a data-swap-method="post" data-swap-action="/swap-method-post" data-swap-body='{"name":"charles"}'>Link Data</a>
+        <a data-swap-method="post" data-swap-action="/swap-method-post" data-swap-body='{"name":"augustine"}' data-swap=".content">Swap Link Data</a>
```

Comment:
Let's put the `data-swap-action` before the method so instead of:

`<button data-swap-method="post" data-swap-action="/swap-method-post" data-swap-body='{"name":"charles"}'>Post Data</button>`

`<button data-swap-action="/swap-method-post" data-swap-method="post" data-swap-body='{"name":"charles"}'>Post Data</button>`

Reply:
And I'd do that anywhere these buttons exist.

Reply:
got it